### ISSUE TITLE
Global install with yarn fails due to the deprecated dependency of forever-monitor@1.1.0

### ISSUE THREAD

**Issue by rbecheras** [FredrikNoren/ungit#967]
```
$ yarn global add ungit
yarn global v1.0.2
warning package.json: License should be a valid SPDX license expression
[1/4] Resolving packages...
warning isarray@2.0.1: Just use Array.isArray directly
[2/4] Fetching packages...
warning Pattern ["options@latest"] is trying to unpack in the same destination "/home/remi/.cache/yarn/v1/npm-options-0.0.6-ec22d312806bb53e731773e7cdaefcf1c643128f" as pattern ["options@>=0.0.5"]. This could result in a non deterministic behavior, skipping.
info fsevents@1.1.2: The platform "linux" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
error forever-monitor@1.1.0: The engine "node" is incompatible with this module. Expected version "0.8.x".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.
```

`forever-monito@1.1.0` requires `node engine = 0.8.0`:

```
$ yarn info forever-monitor@1.1.0
yarn info v1.0.2
warning package.json: License should be a valid SPDX license expression
{ name: 'forever-monitor',
  versions: 
   [ '1.0.0',
     '1.0.1',
     '1.0.2',
     '1.1.0',
     '1.2.0',
     '1.2.1',
     '1.2.2',
     '1.2.3',
     '1.2.4',
     '1.3.0',
     '1.4.0',
     '1.5.0',
     '1.5.1',
     '1.5.2',
     '1.6.0',
     '1.7.0',
     '1.7.1' ],
  'dist-tags': { latest: '1.7.1' },
  version: '1.1.0',
  description: 'Core forever process monitor',
  author: 
   { name: 'Nodejitsu Inc',
     email: 'info@nodejitsu.com' },
  contributors: 
   [ { name: 'Charlie Robbins',
       email: 'charlie@nodejitsu.com' },
     { name: 'Fedor Indutny',
       email: 'fedor.indutny@gmail.com' },
     { name: 'James Halliday',
       email: 'mail@substack.net' },
     { name: 'Bradley Meck',
       email: 'bradley@nodejitsu.com' },
     { name: 'Dominic Tarr',
       email: 'dominic@nodejitsu.com' },
     { name: 'Maciej Małecki',
       email: 'maciej@nodejitsu.com' } ],
  repository: 
   { type: 'git',
     url: 'http://github.com/nodejitsu/forever-monitor.git' },
  keywords: 
   [ 'fault tolerant',
     'sysadmin',
     'tools' ],
  dependencies: 
   { broadway: '0.2.x',
     minimatch: '0.0.x',
     pkginfo: '0.x.x',
     'ps-tree': '0.0.x',
     watch: '0.5.x',
     utile: '0.1.x' },
  devDependencies: 
   { optimist: '0.3.x',
     vows: '0.6.x' },
  main: './lib/index.js',
  scripts: { test: 'vows test/**/*-test.js --spec -i' },
  engines: { node: '0.8.x' },
  dist: 
   { shasum: '439ce036f999601cff551aea7f5151001a869ef9',
     tarball: 'http://npm.sirap.fr/forever-monitor/-/forever-monitor-1.1.0.tgz' },
  maintainers: 
   [ { name: 'indexzero',
       email: 'charlie.robbins@gmail.com' },
     { name: 'avianflu',
       email: 'charlie@charlieistheman.com' } ],
  directories: {} }
Done in 0.58s.
```

But the latest `forever-monitor@1.7.1` ficed that issue with: `engines: { node: '>= 0.8.x' }` 

```
$ yarn info forever-monitor
yarn info v1.0.2
warning package.json: License should be a valid SPDX license expression
{ name: 'forever-monitor',
  versions: 
   [ '1.0.0',
     '1.0.1',
     '1.0.2',
     '1.1.0',
     '1.2.0',
     '1.2.1',
     '1.2.2',
     '1.2.3',
     '1.2.4',
     '1.3.0',
     '1.4.0',
     '1.5.0',
     '1.5.1',
     '1.5.2',
     '1.6.0',
     '1.7.0',
     '1.7.1' ],
  'dist-tags': { latest: '1.7.1' },
  version: '1.7.1',
  description: 'Core forever process monitor',
  author: 
   { name: 'Charlie Robbins',
     email: 'charlie.robbins@gmail.com' },
  contributors: 
   [ { name: 'Charlie Robbins',
       email: 'charlie@nodejitsu.com' },
     { name: 'Fedor Indutny',
       email: 'fedor.indutny@gmail.com' },
     { name: 'James Halliday',
       email: 'mail@substack.net' },
     { name: 'Bradley Meck',
       email: 'bradley@nodejitsu.com' },
     { name: 'Dominic Tarr',
       email: 'dominic@nodejitsu.com' },
     { name: 'Maciej Małecki',
       email: 'maciej@nodejitsu.com' } ],
  repository: 
   { type: 'git',
     url: 'git+ssh://git@github.com/nodejitsu/forever-monitor.git' },
  keywords: 
   [ 'fault tolerant',
     'sysadmin',
     'tools' ],
  dependencies: 
   { broadway: '~0.3.6',
     chokidar: '^1.0.1',
     minimatch: '~3.0.2',
     'ps-tree': '0.0.x',
     utile: '~0.2.1' },
  devDependencies: 
   { optimist: '~0.6.1',
     vows: '~0.7.0' },
  main: './lib/index.js',
  scripts: { test: 'vows test/**/*-test.js --spec -i' },
  license: 'MIT',
  engines: { node: '>= 0.8.x' },
  gitHead: '7d1c4631d831a017b7ae9430585f8b7324f66162',
  bugs: { url: 'https://github.com/nodejitsu/forever-monitor/issues' },
  homepage: 'https://github.com/nodejitsu/forever-monitor#readme',
  maintainers: 
   [ { name: 'indexzero',
       email: 'charlie.robbins@gmail.com' },
     { name: 'jcrugzz',
       email: 'jcrugzz@gmail.com' } ],
  dist: 
   { shasum: '5d820f4a3a78db2d81ae2671f158b9e86a091bb8',
     tarball: 'http://npm.sirap.fr/forever-monitor/-/forever-monitor-1.7.1.tgz' },
  directories: {} }
Done in 0.59s.
```
So an upgrate to the latest should fix the yarn install

**Comment by rbecheras** [comment]
For those having the same issue wanting to get it works, there is a workaround using the `--ignore-engines` option:

    yarn global add ungit --ignore-engines

**Comment by campersau** [comment]
Related: https://github.com/FredrikNoren/ungit/issues/961

Also see https://github.com/FredrikNoren/ungit/pull/964#issuecomment-330700792 for the various `forever-monitor` bugs.

**Comment by campersau** [comment]
This is now fixed in ungit version 1.1.33 (see #971).

**Comment by rbecheras** [comment]
Thanks!
